.github/dependabot.yaml

```
@@ -0,0 +1,25 @@
+version: 2
+updates:
+- package-ecosystem: "github-actions"
+  directory: "/"
+  schedule:
+    interval: "monthly"
+- package-ecosystem: "gomod"
+  directory: "/"
+  schedule:
+    interval: "monthly"
+  groups:
+    k8s.io:
+      patterns:
+      - "k8s.io/*"
+    go.opentelemetry.io:
+      patterns:
+      - "go.opentelemetry.io/*"
```

Could you please add one more grouping, by prometheus package?
```suggestion
      - "go.opentelemetry.io/*"
    prometheus:
      patterns:
      - github.com/prometheus/*
```

---

done